packages/x-date-pickers/src/internals/hooks/useField/useField.ts

```diff
@@ -110,23 +111,40 @@ export const useField = <
   });
 
   const handleInputPaste = useEventCallback((event: React.ClipboardEvent<HTMLInputElement>) => {
-    if (readOnly || selectedSectionIndexes == null) {
+    event.preventDefault();
+
+    if (readOnly) {
+      return;
+    }
+
+    // TODO: Implement single section paste
+    if (
+      selectedSectionIndexes &&
+      selectedSectionIndexes.startIndex === selectedSectionIndexes.endIndex
+    ) {
       return;
     }
```

Comment:
By the way not using `event.preventDefault()` can do the job. But that's for an other PR

Reply:
For a valid date we would still have to update the parsed date and fire `onChange` no ?

Reply:
I will open a PR to test that when this one is merged